package.json

```
@@ -5,15 +5,32 @@
     "@decidim/browserslist-config": "^0.28.1",
     "@decidim/core": "^0.28.1",
     "@decidim/elections": "^0.28.1",
-    "@decidim/webpacker": "^0.28.1"
+    "@decidim/webpacker": "^0.28.1",
+    "@rails/actioncable": "^6.1.7",
+    "codemirror": "^5.65.16",
+    "form-storage": "^1.3.5",
+    "formBuilder": "^3.19.7",
+    "highlight.js": "^11.9.0",
+    "inline-attacher": "^0.0.7",
+    "jsrender": "^1.0.14",
+    "leaflet.featuregroup.subgroup": "^1.0.2",
+    "tom-select": "^2.2.2"
   },
   "version": "0.1.0",
   "packageManager": "yarn@1.22.22+sha1.ac34549e6aa8e7ead463a7407e1c7390f61a6610",
   "devDependencies": {
-    "@decidim/dev": "^0.28.1",
-    "@decidim/eslint-config": "^0.28.1",
-    "@decidim/prettier-config": "^0.28.1",
-    "@decidim/stylelint-config": "^0.28.1"
+    "@babel/eslint-parser": "^7.24.7",
+    "@babel/plugin-transform-classes": "^7.24.7",
+    "@babel/plugin-transform-regenerator": "^7.24.7",
+    "@babel/plugin-transform-runtime": "^7.24.7",
+    "@babel/preset-env": "^7.24.7",
+    "@babel/preset-react": "^7.24.7",
+    "@decidim/dev": "^0.28.0",
+    "@decidim/eslint-config": "^0.28.0",
+    "@decidim/prettier-config": "^0.28.0",
+    "@decidim/stylelint-config": "^0.28.0",
```

Change it, and then run `npm i` again to install the right version of decidim dev packages:

```suggestion
    "@decidim/dev": "^0.28.1",
    "@decidim/eslint-config": "^0.28.1",
    "@decidim/prettier-config": "^0.28.1",
    "@decidim/stylelint-config": "^0.28.1",
```